delphi/app/plugins/graph_plugin.py

```diff
@@ -0,0 +1,152 @@
+from delphi.app import data_analyzer
+from collections import namedtuple
+
+from delphi.app.graph import FlowNetwork
+from delphi.app.pki_model import PKI
+
+
+def find_path(start_node, flow, people):
+    """
+    find_path is a utility function that returns all that nodes that are able to carry flow from a specific source
+    node
+    :param start_node: The source node to calculate paths for
+    :param flow: List of all nodes in the network with positive flow
+    :param people: The amount of people that must be carried from the source node
+    :return: A list of all possible nodes that contribute to viable paths for start_node
+    """
+    path_list = [start_node]
+    for node in path_list:
+        current_node = node
+        for edge_check in flow:
+            if edge_check.start == current_node and edge_check.end != 'ta':
+                if edge_check.flow >= people / 2:
```

Comment:
is this the same people/2 as in graph.find_path()?

Reply:
This is the metric that I am using to decide if a path is viable for a specific source. If half the room can go down a certain path, then its a viable path. This can be changed to be whatever, I just couldn't think of a better metric.. It's probably not ideal.

Reply:
That makes sense